test/servico/Cst12Test.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package servico;
+
+import datamapper.PopulateDB;
+import datamapper.exceptions.NonexistentEntityException;
+import dominio.Aula;
+import dominio.Ausencia;
+import dominio.Professor;
+import java.text.ParseException;
+import java.util.LinkedList;
+import java.util.List;
+import modelo.AusenciaModel;
+import modelo.ProfessorModel;
+import org.joda.time.Interval;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+
+/**
+ *
+ * @author GUSTAVO
+ */
+public class Cst12Test {
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+        PopulateDB.recreateDB("prosub", "root", "");
+
+            PopulateDB.populateUseCaseTest();

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['method def' child have incorrect indentation level 12, expected level should be 8.](https://www.codacy.com/app/calebepb/substituicao/file/2777718878/issues/source?bid=3364462&fileBranchId=3364490#l36)